bedrock/firefox/views.py

```diff
@@ -351,17 +343,17 @@ def show_default_account_whatsnew(version):
 
 
 class FirstrunView(L10nTemplateView):
-    ftl_files_map = {
-        "firefox/firstrun/firstrun.html": ["firefox/firstrun"],
-        "firefox/developer/firstrun.html": ["firefox/developer"],
-    }
+    ftl_files_map = {"firefox/developer/firstrun.html": ["firefox/developer"], "firefox/new/desktop/download.html": ["firefox/new/desktop"]}
```

Comment:
Issue: these strings are for a different template can should be removed?
```suggestion
    ftl_files_map = {"firefox/developer/firstrun.html": ["firefox/developer"]}
```